README.md

```diff
@@ -216,21 +219,22 @@ CDTReplicator *replicator = [replicatorFactory oneWay:pushReplication error:&err
 [replicator start];
 ```
 ```swift
+import CDTDatastore
 do {
-    // Create and start the replicator -- -start is essential!
-    let replicatorFactory = CDTReplicatorFactory(datastoreManager: manager)
 
-    let s = "https://apikey:apipassword@username.cloudant.com/my_database"
-    let remoteDatabaseURL = NSURL(string: s)
     let datastore = try manager.datastoreNamed("my_datastore");
 
     // Replicate from the local to remote database
-    let pushReplication = CDTPushReplication(source: datastore, target: remoteDatabaseURL)
-    let replicator =  try replicatorFactory.oneWay(pushReplication)
+    let remote = URL(string: "https://apikey:apipassword@username.cloudant.com/my_database")!
```

Comment:
Can we have the variable name include that it's for replicating similar to `pushReplication`?